indico/htdocs/sass/modules/_event_management.scss

```diff
@@ -519,3 +519,43 @@ th.i-table {
         }
     }
 }
+
+#track-list {
+    list-style-type: none;
+    padding-left: 0;
+    width: 800px;
+
+    .i-box {
+        padding: 10px 10px 0 10px ;
+        margin-bottom: 10px;
+    }
+
+    .i-box-header{
+        border-bottom: None;
+        margin-bottom: 0;
+    }
+
+    .i-box-content {
+        border-top: 1px solid $gray;
+        padding: 20px 10px 10px 10px;
+        margin-left: -10px;
+        margin-right: -10px;
+    }
+
+    .gradient{
+        position: relative;
+            &:before {
```

Comment:
too much indentation - also, put this at the end of the block instead of having it between attributes